pygmt/datasets/earth_relief.py

```diff
@@ -10,8 +10,8 @@
 from pygmt.src import grdcut, which
 
 
-@kwargs_to_strings(region="sequence")
-def load_earth_relief(resolution="01d", region=None, registration=None):
+@kwargs_to_strings(convert_bools=False, region="sequence")
```

Comment:
I disabled the default value of `convert_bools`, i.e., `True`. Otherwise, `use_strm` will be transformed to be empty if it is `True`.

Reply:
> I disabled the default value of `convert_bools`, i.e., `True`. Otherwise, `use_strm` will be transformed to be empty if it is `True`. 

Yes, it's a know limitation/potential bug of `kwargs_to_strings` (#640).

> By the way, I don't know if it will trigger potential bugs. I also don't understand why we have to transform bool here.

If someone uses `X=True`, we need to pass `-X` to GMT; if someone uses `X=False`, we have to make sure `-X` is not passed to GMT. That's why the conversion is useful.